.github/actions/compute-sha256/action.yml

```diff
@@ -13,19 +13,11 @@ runs:
   using: "composite"
   steps:
     - name: Compute the sha256
+    - id: compute
+    - uses: actions/setup-node@v3
+        with:
+            node_version: "16"
       id: compute
-      shell: bash
```

Comment:
is there a reason we cannot declare the Action as type `nodejs`?
```
runs:
  using: 'node16'
  main: 'dist/index.js'
```

Reply:
The reason was to keep it as `using:composite`. Let me know if it isn't required.

Reply:
not required, we should move it all to a nodejs Action.

Reply:
I have updated it. Thanks